lib/addons/neuron/index.ts

```diff
@@ -0,0 +1,31 @@
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { loadYaml, readYamlDocument } from "../../utils/yaml-utils";
+
+export class NeuronPluginAddOn implements ClusterAddOn {
```

Comment:
No options to deploy? Not even namespace? It is fine if ns should be kube-system, just want to check if anything is reasonable to expose for configuration.

Reply:
On this one, it is straight forward. There may be some optional scheduler which I just saw, that I will do with options in a fast follow-up.

Reply:
Do you mind creating an issue and assigning to you or Riccardo if you want to do a fast follow-up for options

Reply:
Will do it in this PR actually, testing it right now.